script/1_0_2current_en_script/Ciel/Day 2/02_01_CIEL02_1_2.txt

```diff
@@ -32,9 +32,9 @@ Anyway, I think her contribution made for a spectacular lunch.
 "Thanks for the food, it was delicious.
 If the opportunity arises, I will return the favor next time."
 "N-No, it's okay, today was just a coincidence!
-A-Anyway... Oh right! Did you hear the gossip yesterday about Toyama-sensei?"
+A-Anyway... Oh right! Have you heard the gossip that sprung up yesterday about Toyama-sensei?"
 <Page11>
-"About Toyama-sensei? You mean the news from yesterday?
+"About Toyama-sensei? You mean from yesterday's announcement?
```

Comment:
I think this might be spiriting some context into existence. 
> 戸山の話って、昨日のヤツか？
> About Toyama-sensei? Oh, you mean the ones from yesterday?

Reply:
Hmm, some context might need to be tweaked if we do this change though, since "the ones" applied to gossip doesn't sit well. And actually, I'm not sure these first few lines should be hearsay/speculation, since he gets to that later explicitly -
> Y: そ、それより……そうそう！　戸山先生の話、聞いた？
> S: 戸山の話って、昨日のヤツか？
> S: 不幸にあったとか、そういう噂の
>
> Y: A-Anyway... Oh right! Did you hear the news about Toyama-sensei?"
> S: About Toyama-sensei? Oh, you mean from yesterday?
> S: I've heard rumors that something awful happened."